ezcater/recipe
# Fix issue with FocusScope blur elements not propagating correctly

## What did we change?
Removed the `stopPropagation()` call from `FocusElement`'s `onBlur` listener.

## Why are we doing this?
This call has no obvious use and was preventing downstream apps from propagating `onBlur` methods from being called when an element loses focus. `EzModal`, `EzDataInput`, and `EzPopover` use `FocusElement` as a wrapper for accessibility to prevent tabbing outside the focused element. This behavior is desirable, however there is no obvious reason to prevent propagation of events in `onBlur`.

## Testing
- Run `FocusElement` tests to make sure it still works as intended.
- Run tests individually for `EzModal`, `EzField`, and `EzPopover` and manually test in the playroom to make sure component functionality hasn't changed.
- To test that `onBlur` elements will now propagate, verify that the following will console log as expected when tabbing between elements in the playroom on this branch:

  ```js
  <EzModal isOpen>
    <p>hello world</p>
    <input type="text" onBlur={() => console.log('blur')} />
    <input type="button" onBlur={() => console.log('button blur')} value="test" />
  </EzModal>
  ```

## Checklist

- [x] Create a changeset (`yarn changeset`) with a summary of the changes
